Sources/TestSupport/Resources.swift

```diff
@@ -32,27 +34,11 @@ public class Resources: ManifestResourceProvider {
         return toolchain.libDir
     }
 
-  #if os(macOS)
-    public var sdkPlatformFrameworksPath: AbsolutePath {
-        return toolchain.sdkPlatformFrameworksPath!
-    }
-  #endif
-
     let toolchain: UserToolchain
 
     public static let `default` = Resources()
 
     private init() {
-        let binDir: AbsolutePath
-      #if Xcode
-        // Always point to fake toolchain when in Xcode.
-        binDir = AbsolutePath(#file).parentDirectory
-            .parentDirectory.parentDirectory.appending(components: ".build", "debug")
-      #elseif os(macOS)
-        binDir = bundleRoot()
-      #else
-        binDir = AbsolutePath(CommandLine.arguments[0], relativeTo: currentWorkingDirectory).parentDirectory
-      #endif
-        toolchain = try! UserToolchain(binDir)
+        toolchain = try! UserToolchain(destination: Destination.hostDestination())
```

Comment:
We do these special things here to run tests inside Xcode, please ensure the tests work inside Xcode too by generating a project using `Utilities/bootstrap --generate-xcodeproj`